### DISCUSSION TITLE
Nested Query/Select

### DISCUSSION THREAD

**Discussion by objecta** [Intechnity-com/OdooJsonRpcClient#28] [Q&A]
Hi
I have this query
			var stockRepo = new OdooRepository<StockQuantOdooModel>(_odooClient);
			var onlyStockLocs = await stockRepo.Query()
				.Where(s => s.LocationId, OdooOperator.In, _stockLocations)
				.Where(s => s.LastUpdate, OdooOperator.GreaterThanOrEqualTo, since)				
				.ToListAsync();

In the result i get the ProductId and LocationId. Is there any support for creating queries to include the Product and Location in the result, or do I have to make specific queries for the products and locations?

Klaus

**Comment by patricoos** [comment]
Unfortunately odoo doesn't support including. You have to do direct queries for each of the tables
